test/fixtures/academic_terms.yml

```diff
@@ -0,0 +1,44 @@
+one:
```

Comment:
one, two gibi isimler yerine daha anlamlı fixture isimleri seçmelisin.

```
academic_terms(:one)
```

okuyan için pek bir şey ifade etmiyor ama bunu ismi atıyorum 'spring', 'fall' gibi bir şey olsa okuyan anlayabilir:

```
academic_terms(:spring)
```

gibi çağırırız.

ikinci faydası ise bütün fixture'larında one, two, three gibi aynı isme sahip datalar olmamış olur.